Backend/app/dashboard/Lib/src/Storage/Queries/Person.hs

```diff
@@ -271,3 +272,48 @@ instance ToTType' BeamP.Person Person.Person where
         mobileNumberHash = mobileNumber.hash,
         ..
       }
+
+findAllByFromDateAndToDateAndMobileNumberAndStatusWithLimitOffset ::
+  (BeamFlow m r, EncFlow m r) =>
+  Maybe UTCTime ->
+  Maybe UTCTime ->
+  Maybe Text ->
+  Maybe FleetOwnerStatus ->
+  Maybe Int ->
+  Maybe Int ->
+  m [Person]
+findAllByFromDateAndToDateAndMobileNumberAndStatusWithLimitOffset mbFromDate mbToDate mbMobileNumber mbStatus mbLimit mbOffset = do
+  mbMobileNumberDbHash <- traverse getDbHash mbMobileNumber
+  findAllWithOptionsDb
+    [ Se.And
+        ( [ Se.Or
+              [ Se.Is BeamP.verified $ Se.Eq (Just False),
+                Se.Is BeamP.verified $ Se.Eq Nothing
+              ]
+          ]
+            <> [Se.Is BeamP.createdAt $ Se.GreaterThanOrEq (fromJust mbFromDate) | isJust mbFromDate]
+            <> [Se.Is BeamP.createdAt $ Se.LessThanOrEq (fromJust mbToDate) | isJust mbToDate]
+            <> [Se.Is BeamP.mobileNumberHash $ Se.Eq (fromJust mbMobileNumberDbHash) | isJust mbMobileNumber]
+            <> [Se.Is BeamP.rejectedAt $ checkOfNull Nothing | isJust mbStatus]
```

Comment:
This is not correct, because if admin neither approved nor rejected, then we consider this account as `Approved`
Or if admin `Rejected`, and then next time `Approved`, also logic would be wrong
Lets use another check, based on `verified`

```
<> [Se.Is BeamP.verified $ checkStatus | isJust mbStatus]

...
where
   checkStatus =
      case mbStatus of
        Just Rejected -> Se.Eq False
        _ -> Se.not (Se.Eq False) -- Nothing considered here as Approved
```